Subset_Sum/Subset_Sum.cpp

```diff
@@ -0,0 +1,83 @@
+/* Subset Sum Problem: Given an array of elements and a sum,
```

Comment:
Use 4 space indentation.

Reply:
@SakshiShreya I don't see the need of 4 space indentation at the start of comments.  Indentation is provided for clarity at the start of a block.

Reply:
Please clarify the need of 4 space indentation here @SakshiShreya .

Reply:
Actually she meant throughout the file.

Reply:
Done.